pipeline2/calibrate.py

```diff
@@ -0,0 +1,23 @@
+import tkinter as tk
+from capturing_user_input_data import Capture
+from step1_landmarks import run_openface_feature_extraction
+from step2_features import generate_features
+
+calibrate_path = '../train_data2/calibration'
+
+N = 100
+
+root = tk.Tk()
+capture = Capture(root, N, calibrate_path)
```

Comment:
Good idea to wrap it into a class. Let's put Tk inside it as well, to hide as an implementation detail.